android/app/src/main/res/values/ic_launcher_background.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="ic_launcher_background">#1DA1F2</color>
```

Comment:
Please put this in `colors.xml` in the same folder, and remove `light_blue` from `colors.xml` since it's the exact same color and not used anymore.